### PR TITLE
Add auth status

### PR DESCRIPTION
**Pull request by Bishoy-at-pieces**

### CODE REVIEW

**Comment by Bishoy-at-pieces** [comment]
@mark-at-pieces @mason-at-pieces  @shivay-at-pieces would like to suggest a command name here it is :PiecesStatus

**Comment by mark-at-pieces** [comment]
not a bad name potentially :PiecesAccount or PiecesProfile

File: plugin/pieces.vim [hunk 1]
```diff
@@ -1,13 +1,18 @@
...
 
 
 function! RunPiecesStartup()
     try
         call PiecesStartup()
     catch /^Vim\%((\a\+)\)\=:E/
+        call nvim_command('UpdateRemotePlugins')
         echohl WarningMsg
-        echomsg 'Please run :UpdateRemotePlugins'
+        echomsg 'Warning: Please restart Neovim to ensure the Pieces Plugin works correctly.'
```
Comment (mark-at-pieces) [inline plugin/pieces.vim:15]: "Please restart Neovim to apply the latest updates for the Pieces Plugin."

**Comment by shivay-at-pieces** [comment]
> not a bad name potentially :PiecesAccount or PiecesProfile

I like PiecesAccount

**Review by mark-at-pieces — APPROVED** [review]
just left a note on some copy great stuff

**Pull request merged**